corehq/apps/reminders/views.py

```diff
@@ -102,7 +102,7 @@ def page_context(self):
         today = timezone_now.date()
 
         def adjust_next_fire_to_timezone(reminder_utc):
-            return ServerTime(reminder_utc.next_fire).user_time(timezone).done()
+            return ServerTime(reminder_utc.next_fire).user_time(timezone).done().replace(tzinfo=None)
```

Comment:
I cringe a little to see this, even though looking at the PR as a whole I don't have an obviously better option for you. I say I cringe, because it kind of breaks the only sane-keeping standard we have about datetimes and timezones which are that

1. datetimes without timezones == server time == UTC
2. datetimes with timezones == user time (== probably not utc)

I don't know what the best solution is. Maybe we decide that timezones are such a mess and so different between implementations that we want to only ever do timezone conversions in python; in that case, the thing to do would be do the full conversion to a user display string here in python, and pass that through. Or we decide when we're passing data from python to js, we want to pass the underlying data and let js decide how to display it, in which case we should probably use something like [Moment Timezone](http://momentjs.com/timezone/), pass the server time (UTC) plus the user's preferred timezone to js, and then do the conversion and formatting in js.